src/controllers/update/remove.need.to.make.changes.to.entity.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { NextFunction, Request, Response } from "express";
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.REMOVE_NEED_MAKE_CHANGES_PAGE}`);
+    return res.render(config.REMOVE_NEED_MAKE_CHANGES_PAGE, {
+      journey: config.JourneyType.remove,
+      backLinkUrl: `${config.OVERSEAS_ENTITY_PRESENTER_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`,
```

Comment:
In theory, back link here doesn't need to set the JOURNEY_REMOVE_QUERY_PARAM because by this point the 'is_remove=true' flag is set in the session and Mongo JSON, so that screen can pick it up from there (changes will be required for this though...). A discussion required, perhaps.